jsk_perception/CMakeLists.txt

```diff
@@ -246,23 +246,25 @@ add_dependencies(hough_lines              ${PROJECT_NAME}_gencfg ${PROJECT_NAME}
 add_dependencies(rectangle_detector       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
 add_dependencies(color_histogram_sliding_matcher       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
 
+# FIXME: actually, roseus executable is usable only on Linux
+if(GNU)
```

Comment:
can you setup mac machine which I can log in and try to fix?

--
◉ Kei Okada


On Tue, Apr 26, 2016 at 11:44 AM, Kentaro Wada <notifications@github.com>
wrote:

> In jsk_perception/CMakeLists.txt
> <https://github.com/jsk-ros-pkg/jsk_recognition/pull/1605#discussion_r61021773>
> :
>
> > @@ -246,23 +246,25 @@ add_dependencies(hough_lines              ${PROJECT_NAME}_gencfg ${PROJECT_NAME}
> >  add_dependencies(rectangle_detector       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
> >  add_dependencies(color_histogram_sliding_matcher       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
> >
> > +# FIXME: actually, roseus executable is usable only on Linux
> > +if(GNU)
>
> because roseus can be compiled but it segfaults at ros::roseus.
> … <#m_5700209624246446481_>
> -- 和田 健太郎 / Kentaro Wada http://wkentaro.com/ 2016年4月26日 9:06 +0900 に Kei
> Okada<notifications@github.com>は書きました:
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/jsk-ros-pkg/jsk_recognition/pull/1605/files/18164dc33da667b13fd63c7de564444b416968b6#r61021773>
>